implementations/rust/ockam/ockam_api/src/enroll/oidc_service.rs

```diff
@@ -138,7 +142,7 @@ impl OidcService {
             authorization_code.code
         );
         self.request_code(
-            Url::parse("https://account.ockam.io/oauth/token").unwrap(),
+            Url::parse(&format!("{}/oauth/token", Self::authenticator_endpoint())).unwrap(),
```

Comment:
@etorreborre is there any reason to have this URL hardcoded here instead of using the one from `self.provider().token_request_url`?